src/RoadRegistry.BackOffice.UI/init.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-echo "window.wegenregisterVersion=\"$API_VERSION\";" >> /usr/share/nginx/html/config.js
-echo "window.wegenregisterApiEndpoint=\"$API_ENDPOINT\";" >> /usr/share/nginx/html/config.js
+echo "window.wegenregisterVersion=\"$API_VERSION\";" >> /usr/share/nginx/html/config.f84e1103.js
```

Comment:
Isn't this different every build?

Reply:
Nope, think it's a file / content hash. Been stable for a while.

Reply:
What I'd really like is for `parcel` not to mess with this particular file but haven't found a good way of doing that. Once we do, this will be reverted.

Reply:
Yeah, I was just looking at https://github.com/Informatievlaanderen/road-registry/blob/master/src/RoadRegistry.BackOffice.UI/index.html#L17 and noticed it is `config.js` there, dont know why it rewrote that

Reply:
Because that's the way how bundlers generate unique builds - all files, including html, js, css, etc. get rewritten like that.

Reply:
Yeah, I'm just used to bundlers only bundling what is referenced from the starting point (which config.js normally isnt?)

Reply:
That's the beauty of parcel (and obviously down side in this case) - it's opt-out, not opt-in.